### PR TITLE
feat: add events contract

### LINKED ISSUE #21: Create smart contract with events
## Description
- We want an example of a simple smart contract that emits events once an action is performed.
- Feel free to propose the smart contract structure and methods, It doesn't need to be a complex contract, with a couple of events and methods is good.

## Steps
- Create a new smart contract inside the project's corresponding directory.

## References
- https://book.cairo-lang.org/ch14-03-contract-events.html?highlight=events#defining-events

### PR DESCRIPTION
**Pull request by SoarinSkySagar**
- [x] resolves #21 
 - [x] tests written and all tests passing

### CODE REVIEW

**Comment by SoarinSkySagar** [comment]
PTAL @danielcdz

**Comment by danielcdz** [comment]
can you rename the directory from `events` to `simple_contract_with_events`

File: cairo/scripts/events/tests/test_contract.cairo
[the remark this answers is not in the document]
Comment (SoarinSkySagar) [reply cairo/scripts/events/tests/test_contract.cairo:1]: sure ser

**Comment by SoarinSkySagar** [comment]
> can you rename the directory from `events` to `simple_contract_with_events`

all changes done @danielcdz

**Review by danielcdz — APPROVED** [review]
LGTM

**Pull request merged**